tools/dragen-alignment/3.9.3/dragen-alignment__3.9.3.cwl

```diff
@@ -361,13 +361,13 @@ inputs:
     type: int?
     inputBinding:
       prefix: "--Mapper.edit-seed-num"
-  enable_map_align:
-    label: enable map align
```

Comment:
Although --enable-map-align is set to true by default but for clarity, I'd suggest we keep both inputs, `enable_map_align` and `enable_map_align_output`.

Reply:
Same comment for other pipelines as well. I have checked Dragen docx, and it seems enable_map_align and enable_map_align_output are being explicitly called on the command-line. I am keen to follow the same and not rely on what are the default values for these important parameters.

Reply:
> I'd suggest we keep both inputs, enable_map_align and enable_map_align_output.

Not sure I follow. Running `--enable-map-align-output` forces `--enable-map-align` to be turned on.  

But the docs for enable-map-align suggest it is not relevant to us. 

Under MapperOptions, turning on `--enable-map-align` _enables use of BAM input files for mapper/aligner_

Reply:
Where as `--enable-map-align-output` is 'do you want a bam as output?'

Reply:
Okay, so just enforcing the check here for generation of BAM i.e. should it be generated or not. 
Then makes sense. The initial comment in PR description isn't clear enough

```
--enable-map-align is set to true by default when --enable-map-align is set so no changes to outputs, just makes it more consistent with other workflows
```